core/types/system_tx.go

```diff
@@ -22,7 +23,7 @@ func (tx *SystemTx) txType() byte { return SystemTxType }
 func (tx *SystemTx) copy() TxData {
 	return &SystemTx{
 		Sender: tx.Sender,
-		To:     copyAddressPtr(tx.To),
```

Comment:
The doc above says that "copy creates a deep copy". (Though this is also not respected for `BlobTx`.)